config/ember-try.js

```diff
@@ -50,6 +50,22 @@ module.exports = {
       }
     },
     {
+      name: 'ember-lts-2.12',
+      bower: {
+        dependencies: {
+          'ember': 'components/ember#lts-2-12'
```

Comment:
Can't pull lts-2-12 from bower (the [blueprints do this](https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/files/addon-config/ember-try.js#L20-L27)).